depends/packages/native_qt.mk

```diff
@@ -0,0 +1,77 @@
+# Pretty much taken verbatim from Bitcoin Core
```

Comment:
The reason is pretty much that it was already known to work with Gitian due to Bitcoin Core using it. There is [someone working on moving Bitcoin Core to Qt 5.4.2](https://github.com/bitcoin/bitcoin/issues/6267) and that person should open a PR for Bitcoin Core any day now.